.github/workflows/publish.yml

```diff
@@ -1,23 +1,38 @@
-name: Kirby release CI
+name: Publish to npm
 
 on:
-  push:
-    tags:
-      - '^v[0-9]+\.[0-9]+'
+  pull_request:
+    types: closed
```

Comment:
What if a PR does not target master but some other branch?

Reply:
May be an issue - I'll look into it

Reply:
Good point @Sobuno 👍 🤔

Reply:
Fixed? :)